frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -22,6 +29,24 @@ const plugin: Plugin<ConsumedExtensions> = [
       flag: CEPH_FLAG,
     },
   },
+  {
+    type: 'Dashboards/Tab',
+    properties: {
+      id: 'persistent-storage',
+      title: 'Persistent Storage',
+    },
+  },
+  {
+    type: 'Dashboards/Card',
+    properties: {
+      tab: 'persistent-storage',
+      position: GridPosition.MAIN,
+      loader: () =>
+        import(
+          './components/dashboard-page/storage-dashboard/data-resilience/data-resilience' /* webpackChunkName: "data-resilence-card" */
```

Comment:
Please add `ceph` prefix to webpack chunk name (there is also missing `i` in `data-resilence-card"`
```suggestion
          './components/dashboard-page/storage-dashboard/data-resilience/data-resilience' /* webpackChunkName: "webpack-data-resilience-card" */
```